geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -239,6 +241,12 @@ object RichSimpleFeatureType {
     def getEnabledTables: String = userData[String](SimpleFeatureTypes.ENABLED_INDEXES).getOrElse("")
     def setEnabledTables(tables: String): Unit = sft.getUserData.put(SimpleFeatureTypes.ENABLED_INDEXES, tables)
 
+    def isGeomesaUserData: Boolean = userData[String](GEOMESA_USER_DATA_KEY).getOrElse("false").toBoolean
+    def setGeomesaUserData(geomesaUserData: Boolean): Unit = sft.getUserData.put(GEOMESA_USER_DATA_KEY, geomesaUserData.toString)
+
+    def isAllUserData: Boolean = userData[String](ALL_USER_DATA_KEY).getOrElse("false").toBoolean
+    def setAllUserData(allUserData: Boolean): Unit = sft.getUserData.put(ALL_USER_DATA_KEY, allUserData.toString)
```

Comment:
could we rename these methods so that they're clearer? e.g. persistAllUserData or something